.ci/Jenkinsfile_security_cypress

```diff
@@ -0,0 +1,17 @@
+#!/bin/groovy
+
+library 'kibana-pipeline-library'
+kibanaLibrary.load()
+
+kibanaPipeline(timeoutMinutes: 180) {
+  catchError {
+    workers.base(size: 's', ramDisk: false) {
+      kibanaPipeline.bash('test/scripts/jenkins_security_solution_cypress.sh', 'Execute Security Solution Cypress Tests')
+    }
+  }
+
+  if (params.NOTIFY_ON_FAILURE) {
+    slackNotifications.onFailure(channel: '#security-solution-slack-testing')
+    kibanaPipeline.sendMail(to: 'gloria.delatorre@elastic.co')
```

Comment:
You'll have to grab the implementation for `to:` from that other PR if you want to use it https://github.com/elastic/kibana/pull/70216/files#diff-3f3c1e8e334f00526a9e6fa68670d60a